spec/db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Operations/Methods.class/__methods__/available_flavors_spec.rb

```diff
@@ -0,0 +1,112 @@
+require Rails.root.join('db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Operations' \
+                        '/Methods.class/__methods__/available_flavors.rb').to_s
+
+describe ManageIQ::Automate::Cloud::Orchestration::Operations::AvailableFlavors do
+  let(:default_desc_blank) { "<none>" }
+  let(:default_desc) { "<select>" }
+  let(:root_object) { Spec::Support::MiqAeMockObject.new(root_hash) }
+  let(:root_hash) do
+    { 'service_template' => MiqAeMethodService::MiqAeServiceServiceTemplate.find(service_template.id) }
+  end
+  let(:ems) do
+    @flavor1 = FactoryGirl.create(:flavor, :name => 'flavor1')
```

Comment:
@pkomanek could the flavor1 and flavor2 also be lets instead of instance variable